azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -160,6 +163,39 @@ private void loadPluginJobTypes(final JobTypePluginSet plugins)
     plugins.setCommonPluginJobProps(commonPluginJobProps);
     plugins.setCommonPluginLoadProps(commonPluginLoadProps);
 
+    if (commonPluginJobProps
+        .containsKey(Constants.PluginManager.DEFAULT_PROXY_USERS_JOBTYPE_CLASSES)) {
+      plugins.addDefaultProxyUsersJobTypeClasses(commonPluginJobProps
+          .getStringList(Constants.PluginManager.DEFAULT_PROXY_USERS_JOBTYPE_CLASSES));
+    }
+
+    if (commonPluginJobProps.containsKey(PluginManager.DEFAULT_PROXY_USERS_FILTER)) {
+      plugins.addDefaultProxyUsersFilter(
+          commonPluginJobProps.getStringList(PluginManager.DEFAULT_PROXY_USERS_FILTER));
+    }
+
+    // Loads the default-proxy-users mappings for all job types.
+    final File defaultProxyUsersFile = new File(jobPluginsDir,
+        Constants.PluginManager.DEFAULT_PROXY_USERS_FILE);
+    if (defaultProxyUsersFile.exists()) {
+      LOGGER.info("Default proxy users file " + defaultProxyUsersFile
+          + " found. Attempt to load.");
```

Comment:
Can we change "Attempt to load" to "Attempting to load"?

Reply:
done